web/app/scripts/navbar/navbar-controller.js

```diff
@@ -3,10 +3,12 @@
 
     /* ngInject */
     function NavbarController($log, $window, $rootScope, $scope, $state,
-                              GeographyState, RecordState, BoundaryState, WebConfig) {
+                              BoundaryState, GeographyState, InitialState,
+                              RecordState, WebConfig) {
         var ctl = this;
-        var _ = $window._;
-        init();
+        var initialized = false;
```

Comment:
Yeah I think it can be deleted https://github.com/WorldBank-Transport/DRIVER/pull/77#discussion_r37782812